### PR TITLE
Fix invisible camera viewfinder when no posteffects

### LINKED ISSUE #45: Disabling Post-processing effects on cameras via config file disables camera preview
Steps to reproduce:
- Edit your Open Brush.cfg config file to disable `PostEffectsOnCapture`. Example:
```json
{
  "User": {
  },
  "Video": {
  },
  "Flags": {
    "PostEffectsOnCapture" : false,
  },
  "Export": {
  },
}
```
- Load Open Brush.
- Activate the camera tool.

Expected Result: Can see camera preview on each tool, with post-processing effects disabled.
Actual Result: No preview visible.
Platforms reproduced on: Windows 10, Unity editor, Release and Experimental built versions.

image

### PR DESCRIPTION
**Pull request by mikeskydev**
Fixes #45, at least in the viewfinder. This may be part of a bigger set of issues related to the postprocessing shader stack, but this will restore the viewfinder functionality for now!

### CODE REVIEW

File: Assets/Shaders/SnapshotCameraFlash.shader [hunk 1]
```diff
@@ -73,6 +73,7 @@ Shader "Unlit/Snapshot Camera Flash"
                 col.a *= _Alpha;
                 // apply fog
                 UNITY_APPLY_FOG(i.fogCoord, col);
+                col.a = 1.0;
```
Comment (mikeage) [inline Assets/Shaders/SnapshotCameraFlash.shader:76]: Can you remove line 73?

Comment (mikeskydev) [reply Assets/Shaders/SnapshotCameraFlash.shader:76]: Yes, also I'd like to consider this a temporary fix until we can look at the shader (and other screenshot shaders) properly :D

Comment (mikeage) [reply Assets/Shaders/SnapshotCameraFlash.shader:76]: makes sense. Need any testing? Just approval?

Comment (mikeskydev) [reply Assets/Shaders/SnapshotCameraFlash.shader:76]: Would be great if someone can confirm this hotfix works, I'm not sure if/how config works on quest either, I've only tested against desktop.

**Pull request merged**